[Portland 2023] Add CFP (#1847)
writethedocs/www: docs/conf/portland/2023/cfp.rst

Call for Proposals
==================

> Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next **in-person** {{ city }} conference, coming up on {{date.main}}.

Reviewer: ```suggestion
It's that time of year again: We’re now accepting talk proposals for our next **in-person** {{ city }} conference, coming up on {{date.main}}.
```

Reviewer: Just simplifying a bit :)

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience. Whether you've worked on documentation for decades or you've just started this year, we want to hear from you!
Read on to learn more about the goals of the conference and what we look for in talk proposals.

{% if flagcfp %}
In the meantime, mark your calendars:

**The deadline for submitting proposals is 11:59 PM {{tz}} on {{cfp.ends}}.**

> We'll let you know if your proposal has been accepted by the end of {{cfp.notification}}.

Reviewer: - Can we make the pretalx link open in a new tab? I tried adding the suggestion directly here but because the text wasn't changed it won't let me.
- Ran through the application process, LGTM! Just a question, since this is the first in-person in a while, do we want to add another checkbox after the CoC that says something like "I confirm that I can attend the conference if accepted and will let you know ASAP if I get sick or can't travel" or similar? I know we mention it on the website, but having it right there blocking the submission button might help folks who don't read the entire CFP page or blog post...

Author: Nice call on the confirmation. No new tab / window of the box.

Author: ![image](https://user-images.githubusercontent.com/95874/203558312-113256e6-a024-4800-ad7a-cc53225c2ed8.png)

Author: Of course new tab possible, fixed.

Call for Proposals
==================

It's that time of year again: We’re now accepting talk proposals for our next **in-person** {{ city }} conference, coming up on {{date.main}}.

> (If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for `Atlantic`_ held online in a timezone as widely accessible as possible).

Reviewer 2: ```suggestion
(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for `Atlantic`_ held online between the CEST and EDT timezones).
```

    <div class="announcement" style="background-color:white;">
        <div class="uk-container">
        <a style="border-bottom: none; font-size: .875rem;" class="uk-button uk-button-announcement uk-text-center" href="{{ cfp.url }}" target="_blank">Submit your proposal</a>
        </div>
    </div>

{% endif %}

.. _Atlantic:

Write the Docs Atlantic, online September 2023
----------------------------------------------

If you want to speak at Write the Docs, but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, coming in September 2023.

> This conference will entirely virtual, in a timezone that works for as many people as possible, so you canm present or attend from anywhere.

Reviewer 2: ```suggestion
This conference will entirely virtual, between the CEST and EDT timezones, so you can present or attend from anywhere.
```
